### DISCUSSION TITLE
How to use RootRouteBuilder

### DISCUSSION THREAD

**Discussion by ka2n** [uhyo/rocon#35] [Q&A] [answered]
Hello.

I would like to try to use `RootRouteBuilder`, but I could not get it to work.
Is there any mistake in my code?

https://codesandbox.io/s/eloquent-darkness-0ci2xs?file=%2Fsrc%2FApp.tsx

**Comment by uhyo** [comment, marked as answer]
Thank you for your interest. Your code doesn't look anything wrong. Unfortunately this seems to be a bug of `RootRouteBuilder` 🥲 
I will work on a fix.

**Reply by uhyo** [reply]
#36

**Reply by uhyo** [reply]
@ka2n Thanks for your patience, upgrading to `rocon@1.2.7` should solve your problem. I tested by myself: https://codesandbox.io/s/quirky-shockley-tmqo1k

**Reply by ka2n** [reply]
Thank you!!
